src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -1001,6 +1045,28 @@ export default {
 				this.getTributeContainer().classList.remove(this.$style['tribute-container--focus-visible'])
 			}
 		},
+
+		/**
+		 * Show tribute menu programmatically. If using a click event, stop propagation to prevent closing the menu
+		 * @param {string} trigger - trigger character, can be '/', '@', or ':'
+		 *
+		 * @public
+		 */
+		showTribute(trigger) {
+			this.focus()
+			const index = this.tribute.collection.findIndex(collection => collection.trigger === trigger)
+			this.tribute.showMenuForCollection(this.$refs.contenteditable, index)
+			document.addEventListener('click', this.hideTribute)
```

Comment:
What about using a capturing event here? Then we don't need a `@click.stop`:

```suggestion
document.addEventListener('click', this.hideTribute, true)
```